web/public/static/branding/templates/gateway/style.css

```
@@ -0,0 +1,69 @@
+    display: grid;
+    grid-template-columns: minmax(0, 1.05fr) minmax(0, 0.95fr);
+    gap: clamp(2rem, 6vw, 3.6rem);
+    align-items: stretch;
+    isolation: isolate;
+}
+
+body[data-portal-template="gateway"] [data-portal-role="root"]::before {
+    content: "";
+    position: absolute;
+    inset: clamp(1.5rem, 3vw, 2.4rem);
+    inset-right: auto;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)